engine/src/main/java/org/hibernate/search/engine/search/predicate/dsl/CommonQueryStringPredicateOptionsStep.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.predicate.dsl;
+
+import org.hibernate.search.engine.search.common.BooleanOperator;
+
+/**
+ * The final step in an "abstract" "query string" predicate definition, where optional parameters can be set.
```

Comment:
I'd go with simply `a query string predicate definition`. "abstract" is a bit confusing here.